.travis.yml

```diff
@@ -6,7 +6,7 @@ node_js:
 sudo: true
 
 env:
-  - CXX=g++-4.8 NODE_ENV=testing ALBUMS_API_URL=https://jsonplaceholder.typicode.com
+  - CXX=g++-4.8 NODE_ENV=testing ALBUMS_API_URL=https://jsonplaceholder.typicode.com EXPIRATION_TIME=4
```

Comment:
You can separate env variables in different lines. Check this out `https://docs.travis-ci.com/user/environment-variables/`.